ui/src/app/components/start/AdventurerForm.tsx

```diff
@@ -449,13 +449,13 @@ export const AdventurerForm = ({
           <div className="flex flex-col h-full">
             <Image
               className="mx-auto border border-terminal-green absolute  object-cover py-4 px-8"
-              src={"/monsters/starterbeast.png"}
+              src={"/scenes/intro/beast.png"}
               alt="adventurer facing beast"
               fill
             />
 
             {!isWrongNetwork && (
-              <div className="absolute top-6 left-0 right-0 sm:p-4 text-xs sm:text-xl leading-loose z-10 text-center">
```

Comment:
Removing this makes the text off centre for my screen
<img width="1725" alt="Screenshot 2023-08-15 at 21 53 08" src="https://github.com/BibliothecaDAO/loot-survivor/assets/42612612/3882ab3c-cfd4-45e7-885d-692b27e7a141">